### DISCUSSION TITLE
剪贴板问题

### DISCUSSION THREAD

**Discussion by ZGGSONG** [he3-app/he3-feedback#288] [Bug Report]
1. 打开HE3以后我自己写的软件在写入剪贴板时会崩溃，原因是HE3占用了剪贴板

```txt
# exteption content
System.Runtime.InteropServices.Comexception openclipboard 失败
```
当然我自己这边解决了问题。但是同样是剪贴板工具，Ditto 却不会导致这样的问题，是否需要优化一下

2. 设置页面设置剪贴板预览器关闭，但是重启后UI右上角仍然显示剪贴板预览器，是否为BUG

**Comment by lyzhang1999** [comment]
感谢反馈，已经创建了 issues 跟进：https://github.com/he3-app/he3-feedback/issues/290
